bundles/org.openhab.io.homekit/README.md

```diff
@@ -520,6 +524,17 @@ Number          thermostat_cool_thrs       "Thermostat Cool Threshold Temp [%.1f
 Number          thermostat_heat_thrs       "Thermostat Heat Threshold Temp [%.1f °C]" (gThermostat) {homekit = "HeatingThresholdTemperature"}
 ```
 
+If your thermostat simply has a heating set point and cooling set point, and uses those points regardless of HEAT, COOL, or AUTO mode, you may simply omit the TargetTemperature characteristic, and the add-on will dynamically route to those characteristics as appropriate:
+
+```xtend
```

Comment:
Please use java for item code fences:
For reference: https://github.com/openhab/openhab-docs/issues/999